_tutorials/2016-05-16-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,77 @@
+---
+published: true
+date: "2016-05-16 09:08 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64"
+author: "Lisa Roach, Ric Wellum"
```

Comment:
Oh gotcha! I just wanted to give Ric credit. I'll try it with one author.

Thank you,
Lisa

Sent from my iPhone

On May 16, 2016, at 9:19 PM, Akshat Sharma <notifications@github.com<mailto:notifications@github.com>> wrote:


In _tutorials/2016-05-16-getting-started-on-xrv64.md<https://github.com/xrdocs/application-hosting/pull/17#discussion_r63462611>:

> @@ -0,0 +1,77 @@
> +---
> +published: true
> +date: "2016-05-16 09:08 -0700"
> +title: Getting Started on XRv64
> +permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64"
> +author: "Lisa Roach, Ric Wellum"


One author per tutorial, I'm afraid. Lisa, since you've been added to the database, use "Lisa Roach" in the Author field

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/xrdocs/application-hosting/pull/17/files/9c23a7e45eb3f8431261fbd055fb356234f3976c#r63462611>